### PR TITLE
fix(testing): remove datetimepicker extraneous logs

### PR DESCRIPTION
**Pull request by ocBruno**
add date-fns for locales, import and register enUS and ptBR localeObjects, set defaultLocale to enUS
and update test to match.

@jackcmeyer I think this is the fix for this issue:
https://github.com/HospitalRun/hospitalrun/issues/146
but I wasn't sure since it was in the main repo issues. 
If anything it fixes these extraneous logs in the components library

**Newly added dependencies with [Bundlephobia](https://bundlephobia.com/) links:**
- "date-fns": "^2.9.0",
https://bundlephobia.com/result?p=date-fns@2.9.0

### CODE REVIEW

**Comment by ocBruno** [comment]
fix #251 
After moving to the other repo the issue lost the link.

File: src/components/DateTimePicker/DateTimePicker.tsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
 import React, { ReactNode } from 'react'
-import DatePicker from 'react-datepicker'
+import DatePicker, { registerLocale, setDefaultLocale } from 'react-datepicker'
 import InputGroup from 'react-bootstrap/InputGroup'
+import { enUS, ptBR } from 'date-fns/locale'
 import { Icon } from '../Icon'
 import 'react-datepicker/dist/react-datepicker.css'
 
+registerLocale('enUS', enUS)
+registerLocale('ptBR', ptBR)
```
Comment (fox1t) [inline src/components/DateTimePicker/DateTimePicker.tsx]: Why adding just BR locale? @ocBruno we might open a new issue to discuss about which locales to support. What do you think?

Comment (ocBruno) [reply src/components/DateTimePicker/DateTimePicker.tsx]: Definetly agree @fox1t, I had began adding the rest of them but then I wasn't exactly sure  considering the language differences of the same place of origins and how many to add. 
I think depending on the language difference the resources will need to updated accordingly, so adding language support for en-us en-gb might not be worth it for the time being considering the small differences but other languages such as pt-BR pt-PT vary a bit more.

**Comment by fox1t** [comment]
LGTM

**Pull request merged**